job.go

```diff
@@ -1,15 +1,19 @@
 package river
 
 import (
-	"context"
-	"encoding/json"
-	"errors"
-	"time"
-
-	"github.com/riverqueue/river/riverdriver"
 	"github.com/riverqueue/river/rivertype"
 )
 
+const (
+	JobStateAvailable = rivertype.JobStateAvailable
+	JobStateCancelled = rivertype.JobStateCancelled
+	JobStateCompleted = rivertype.JobStateCompleted
+	JobStateDiscarded = rivertype.JobStateDiscarded
+	JobStateRetryable = rivertype.JobStateRetryable
+	JobStateRunning   = rivertype.JobStateRunning
+	JobStateScheduled = rivertype.JobStateScheduled
+)
```

Comment:
Also hoisted these job state definitions to the top of the file. Feels a little weird that they lived at the bottom.